### PR TITLE
jason-improve-code

### PR DESCRIPTION
**Pull request by JasonScourge**

### CODE REVIEW

**Comment by JasonScourge** [comment]
Still checking the coverage/coveralls

**Comment by JasonScourge** [comment]
Error remedied.

File: src/main/java/seedu/doerList/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -88,6 +88,7 @@ public CommandResult execute() {
 
 	}
 
+	//@@author A0139401N
```
Comment (xpdavid) [inline src/main/java/seedu/doerList/logic/commands/EditCommand.java]: @JasonScourge This is not done by you... @haiilong first got the `generateUpdatedTask` idea.

Comment (JasonScourge) [inline src/main/java/seedu/doerList/logic/commands/EditCommand.java]: Placed wrongly. Remedied.

**Pull request merged**